### DISCUSSION TITLE
Localhost:3000 is throwing me errors when trying to view blog

### DISCUSSION THREAD

**Discussion by nballon3** [vercel/next.js#31257] [Help]
Good morning. I am new to Sanity.io and Next.js so please bare with me. I went through the steps to set up my local environment for the demo blog and I got the Studio to show up and it is fully functional but when I try to launch the actual blog on my 3000 port, I get two errors thrown. I am attaching the screenshots now. 

Any help would be welcomed. This seems to be the last hurdle for me in regards to working locally. I can also share my repo if that would help resolve the matter.

Thanks in advance.

Noel
![server-error]
(https://user-images.githubusercontent.com/43190186/141144022-efc3dc94-9b3d-4851-8c25-24ff7ddb6a58.png)

![unhandled-runtime-error]
(https://user-images.githubusercontent.com/43190186/141144025-e7600d93-c0b7-44c3-8cf1-634f146b0fac.png

**Comment by delbaoliveira** [comment]
Hi @nballon3,

Could you share the repo? Would be happy to take a look. You can DM me on [Twitter](https://twitter.com/delba_oliveira) if you prefer not to share it publicly.

**Reply by nballon3** [reply]
Hello Delba, did you receive my Twitter message?

**Reply by nballon3** [reply]
Hi @delbaoliveira were you able to take a look at my issue?

**Reply by delbaoliveira** [reply]
Hi @nballon3, I've been waiting for you to respond to my Twitter message. Would you like to continue our conversation here or on Twitter?

**Comment by nballon3** [comment]
Oh ok, I never got aby notification from Twitter. Let'd keep here so I have a trail of our comminication I can later share with my team manager.Sent from my T-Mobile 5G Device
-------- Original message --------From: Delba de Oliveira ***@***.***> Date: 11/16/21  5:24 AM  (GMT-07:00) To: "vercel/next.js" ***@***.***> Cc: nballon3 ***@***.***>, Mention ***@***.***> Subject: Re: [vercel/next.js] Localhost:3000 is throwing me errors when trying to view blog (Discussion #31257) CAUTION: This email originated from outside your organization. Exercise caution when opening attachments or clicking links, especially from unknown senders.


Hi @nballon3, I've been waiting for you to respond to my Twitter message. Would you like to continue our conversation here or on Twitter?

**Reply by nballon3** [reply]
OK, you now are a collaborator on my local repo  @delbaoliveira. Let me know if it did not work.

**Reply by nballon3** [reply]
Now I am getting 2 different errors when compiling locally -
./node_modules/next/dist/compiled/css-loader/cjs.js??ruleSet[1].rules[2].oneOf[6].use[1]!./node_modules/next/dist/compiled/postcss-loader/cjs.js??ruleSet[1].rules[2].oneOf[6].use[2]!./styles/index.css
TypeError: getProcessedPlugins is not a function

and this one -
error - ./node_modules/next/dist/compiled/css-loader/cjs.js??ruleSet[1].rules[2].oneOf[6].use[1]!./node_modules/next/dist/compiled/postcss-loader/cjs.js??ruleSet[1].rules[2].oneOf[6].use[2]!./styles/index.css
TypeError: getProcessedPlugins is not a function
